### PR TITLE
[Cloud PC] Add windowsSettings to cloudPcProvisioningPolicy

### PR DESCRIPTION
**Pull request by RuiHou105**

### CODE REVIEW

**Comment by RuiHou105** [comment]
**Roadmap workitem:**

Feature 12472: [Devices and apps][Cloud PC] Add windowsSettings to cloudPcProvisioningPolicy (Currently hide)

**Comment by Lauragra** [comment]
@RuiHou105 , is this PR ready for content review? The schema associated with this PR has already been released. Please set the **ready for content review** label when you're ready so we can get the docs reviewed and published. Thanks.

File: api-reference/beta/api/cloudpcprovisioningpolicy-update.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ The following table shows the properties that can be updated for the [cloudPcPro
 |imageId|String|The ID of the OS image you want to provision on Cloud PCs. The format for a gallery type image is: {publisher_offer_sku}. Supported values for each of the parameters are as follows: <ul><li>publisher: Microsoftwindowsdesktop.</li> <li>offer: windows-ent-cpc.</li> <li>sku: 21h1-ent-cpc-m365, 21h1-ent-cpc-os, 20h2-ent-cpc-m365, 20h2-ent-cpc-os, 20h1-ent-cpc-m365, 20h1-ent-cpc-os, 19h2-ent-cpc-m365 and 19h2-ent-cpc-os.</li></ul>|
 |imageDisplayName|String|The display name for the OS image you’re provisioning.|
 |imageType|cloudPcProvisioningPolicyImageType|The type of OS image (custom or gallery) you want to provision on Cloud PCs. Possible values are: `gallery`, `custom`.|
+|windowsSettings|[cloudPcWindowsSettings](../resources/cloudpcwindowssettings.md)|Specific windows settings to configure while creating Cloud PCs for this provisioning policy.|
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpcprovisioningpolicy-update.md:60]: Could you help update this to "The Windows operation system settings for the provisioned Cloud PCs with this provisioning policy. Such as operation system language setting."

**Review by AshleyYangSZ — APPROVED** [review]

File: api-reference/beta/api/cloudpcprovisioningpolicy-update.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/api/cloudpcprovisioningpolicy-update.md]: Updated.

File: api-reference/beta/resources/cloudpcwindowssettings.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: "cloudPcWindowsSettings resource type"
+description: "Represents specific windows settings to configure while creating Cloud PCs for a provisioning policy."
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpcwindowssettings.md:3]: Global comment: Please apply this feedback to all the files attached to the PR. 
Please capitalize the word Windows.

File: api-reference/beta/resources/cloudpcwindowssettings.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+author: "RuiHou105"
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpcwindowssettings.md:4]: Please double-check whether the author should be Bhavya Chopra or not.

File: api-reference/beta/resources/cloudpcwindowssettings.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+doc_type: resourcePageType
+---
+
+# cloudPcWindowsSettings resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents specific windows settings to configure while creating Cloud PCs for a provisioning policy.
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpcwindowssettings.md:16]: Global comment: Please apply this feedback to all the files attached to the PR.
Please capitalize the word Windows.

File: api-reference/beta/api/cloudpcprovisioningpolicy-get.md [hunk 5]
```diff
@@ -124,8 +124,11 @@ Content-Type: application/json
     "id": "1d164206-bf41-4fd2-8424-a3192d39ffff",
     "imageDisplayName": "Image Display Name value",
     "imageId": "Image ID value",
-    "imageType":"custom",
-    "onPremisesConnectionId": "4e47d0f6-6f77-44f0-8893-c0fe1701ffff"
+    "imageType": "custom",
```
Comment (Danielabom) [inline api-reference/beta/api/cloudpcprovisioningpolicy-get.md:127]: Quick question: Is there a reason why the other operations for cloudprovisioningpolicy (list, create, delete) are not attached to the PR?

File: api-reference/beta/resources/cloudpcwindowssettings.md (hunk 4 above)
Comment (RuiHou105) [reply api-reference/beta/resources/cloudpcwindowssettings.md]: @Danielabom  I updated all `windows `to `Windows `in this PR. Please check. Thanks!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**